### PR TITLE
fix: Output keyword ordering

### PR DESCRIPTION
**Pull request by Aleksander-Karlsson**
Description
Include a description of the change done
Attempt to reorder the output created by Completor to match the input.

Why
Older versions of Completor moves the modified keywords into one section, thus disregarding the order of the original input schedule. This can cause problems and Completor should respect the original sorting of keywords.

# Checklist:
Before submitting this PR, please make sure:

- [x] I am complying with the [contributing](https://equinor.github.io/completor/contribution_guide) doc
- [x] My code builds clean without any errors or warnings
- [x] I have added/extended tests that prove my fix is effective or that my feature works
- [x] I have made corresponding changes to the documentation, and it builds correctly

### CODE REVIEW

File: completor/create_output.py [hunk 1]
```diff
@@ -10,44 +10,41 @@
...
-        paths: Paths of the case and schedule files. Defaults to None.
 
     Returns:
-        Properly formatted output string ready to be written to file.
-
+        Properly formatted output data divided into relevant keywords + bonus.
     """
     output = []
 
     if case.completion_table[Headers.DEVICE_TYPE].isin([Content.AUTONOMOUS_INFLOW_CONTROL_VALVE]).any():
```
Comment (hrsut) [inline completor/create_output.py:37]: We can remove this, no use on this `CORRELATION_UDQ`

File: completor/main.py [hunk 2]
```diff
@@ -179,116 +179,69 @@ def create(
...
+        for chunk in find_keyword_data(Keywords.COMPLETION_SEGMENTS, schedule):
+            clean_data = clean_raw_data(chunk, Keywords.COMPLETION_SEGMENTS)
+            meaningful_data = read_schedule.set_compsegs(meaningful_data, clean_data)
+
+        for i, well_name in tqdm(enumerate(active_wells.tolist()), total=len(active_wells)):
+            well = Well(well_name, i, case, meaningful_data[well_name])
+            compdat, welsegs, compsegs, bonus = create_output.format_output(well, case, figure_name)
+
+            for keyword in [Keywords.COMPLETION_SEGMENTS, Keywords.WELL_SEGMENTS, Keywords.COMPLETION_DATA]:
+                # TODO: Move to utils? parsing?
```
Comment (hrsut) [inline completor/main.py:213]: I guess this one makes it fail in the CI?

**Review by hrsut — APPROVED** [review]

File: completor/main.py (hunk 2 above)
Comment (Aleksander-Karlsson) [reply completor/main.py:132]: Yes! I forgot to remove it!

Comment (Aleksander-Karlsson) [reply completor/main.py]: Yes! Removing, it's already done, just forgot about the actual TODO:

**Review by MegaAnda123 — APPROVED** [review]

**Pull request merged**